web-server/v0.4/package.json

```diff
@@ -50,7 +50,9 @@
     "react-document-title": "^2.0.3",
     "react-dom": "^16.8.3",
     "react-fittext": "^1.0.0",
+    "react-redux": "^7.1.3",
```

Comment:
Is there new functionality being offered by upgrading/including these dependencies? If so, we should make sure this doesn't break existing functionality in the dashboard.

Reply:
Removed!